src/plugins/inputs/otr.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function(network, chan, cmd, args) {
+	if (cmd !== "otr") {
+		return;
+	}
+	if (args.length === 0 || args[0] === "") {
+		return;
+	}
+	var subCmd = args[0];
+	var client = this;
+	var irc = network.irc;
+	var otrStore = client.otrStore;
+	var session = otrStore.getSession(chan.title, network);
+
+	if (subCmd === "init") {
```

Comment:
There will be at least 4 commands at the end (init/finish/trust/untrust), so nested if is not a good option IMHO.